autoload/wiki.vim

```diff
@@ -162,6 +162,7 @@ function! s:init_buffer_commands() abort " {{{1
   command! -buffer WikiGraphFindBacklinks call wiki#graph#find_backlinks()
   command! -buffer WikiGraphIn            call wiki#graph#to_current()
   command! -buffer WikiGraphOut           call wiki#graph#from_current()
+  command! -buffer WikiJournalIndex       call wiki#journal#make_index()
```

Comment:
Ok, so, the last part is to add the argument to the command. Either two versions of the command, or perhaps better, a "bang" command, i.e. something like this:

```vim
command! -buffer -bang WikiJournalIndex call wiki#journal#make_index(<q-bang> == '!')
```

This also requires a minor expl. in the docs, as well as two versions of the `<plug>` mapping.

Reply:
I.e., `WikiJournalIndex` uses wiki type links, and `WikiJournalIndex!` uses markdown links.

Reply:
Ah! Right, sorry I missed this. One moment